clean MEV-Geth intro (#60)
flashbots/docs: docs/flashbots-core/miners/advanced/interacting-with-mev-relay.md

title: interacting with mev-relay
---

## v0.1

In order to facilitate your interaction with MEV-Relay, we have created the mev-proxy repository as a simple example of a reserve proxy a miner can run to expose just the eth_sendBundle JSON-RPC method.

Access the mev-proxy repository [here](https://github.com/flashbots/mev-proxy).

## v0.2

> Starting from version v0.2 it is possible to use WebSockets to connect to MEV relay. See more details [here](../mev-geth-spec/v02.md)

Reviewer: should be changed from 'it is possible' to 'must'

> Starting from version v0.2 it is possible to use WebSockets to connect to MEV relay. See more details [here](../mev-geth-spec/v02.md)

Reviewer: ```suggestion
Starting from version v0.2 miners must use WebSockets to connect to MEV relay. See more details [here](../mev-geth-spec/v02.md)
```
